NitroxModel-Subnautica/DataStructures/GameLogic/Buildings/Rotation/Metadata/BaseFaceRotationMetadata.cs

```diff
@@ -0,0 +1,42 @@
+﻿using ProtoBufNet;
+using System;
+using NitroxModel.DataStructures.GameLogic.Buildings.Rotation;
+using NitroxModel_Subnautica.Helper.Int3;
+
+namespace NitroxModel_Subnautica.DataStructures.GameLogic.Buildings.Rotation
+{
+    [Serializable]
+    [ProtoContract]
+    public class BaseFaceRotationMetadata : RotationMetadata
+    {
+        public const long VERSION = 1;
+
+        // Base modules anchor based on a face.  This can be constructed via these two attributes.
+        [ProtoMember(1)]
+        public NitroxModel.DataStructures.Int3 AnchoredFaceCell { get; set; }
+
+        [ProtoMember(2)]
+        public int AnchoredFaceDirection { get; set; }
+
+        [ProtoMember(3)]
+        public int AnchoredFaceType { get; set; }
+
```

Comment:
minor: if all faces are an anchored face then we may not need to prefix all with anchored.  Same in file below.

Reply:
I just wanted to make it a little bit clearer for what these members are needed in a RotationMetadata-Class.